lighthouse-core/lib/page-functions.js

```diff
@@ -145,10 +145,10 @@ function ultradumbBenchmark() {
 }
 
 module.exports = {
-  wrapRuntimeEvalErrorInBrowser,
-  registerPerformanceObserverInPage,
-  checkTimeSinceLastLongTask,
-  getElementsInDocument,
-  getOuterHTMLSnippet,
-  ultradumbBenchmark,
+  wrapRuntimeEvalErrorInBrowserString: wrapRuntimeEvalErrorInBrowser.toString(),
+  registerPerformanceObserverInPageString: registerPerformanceObserverInPage.toString(),
+  checkTimeSinceLastLongTaskString: checkTimeSinceLastLongTask.toString(),
+  getElementsInDocumentString: getElementsInDocument.toString(),
+  getOuterHTMLSnippetString: getOuterHTMLSnippet.toString(),
+  ultradumbBenchmark: ultradumbBenchmark,
```

Comment:
this one is still toString()d within driver. can we update this one as well?

Reply:
I had him change it back because we use the raw function in the scripts

Reply:
why not both?

Reply:
.gif

Reply:
![both](https://media2.giphy.com/media/zbzNUbpFnlw8E/giphy.gif)

Reply:
Doesn't this break https://github.com/GoogleChrome/lighthouse/blob/24a855d325617c3cf330929716c3270dbe165430/lighthouse-core/scripts/benchmark.js#L18 ?

Reply:
Nah. It's still exposed the same way. It's just ALSO being exposed as a string.

patrick didnt like my idea of only exposing as string and `eval`ing in the script/benchmark.js. :) haha